js/components/navbar.js

```diff
@@ -1,6 +1,8 @@
-import { urls } from "../shared.js";
+import { urls , isParticipantDataDestroyed } from "../shared.js";
 
 export const userNavBar = (data) => {
+    const newClass = isParticipantDataDestroyed(data.data) ? 'disabled': '';
```

Comment:
would a more descriptive name for `newClass` be helpful for someone editing in the future?